src/scad_clj/scad.clj

```diff
@@ -73,12 +76,18 @@
    (mapcat #(write-expr (+ depth 1) %1) block)
    (list (indent depth) "}\n")))
 
-(defmethod write-expr :rotate [depth [form [a [x y z]] & block]]
+(defmethod write-expr :rotatev [depth [form [a [x y z]] & block]]
   (concat
    (list (indent depth) "rotate (a=" (/ (* a 180) pi) ", v=[" x ", " y ", " z "]) {\n")
    (write-block depth block)
    (list (indent depth) "}\n")))
 
+(defmethod write-expr :rotatec [depth [form [x y z]] & block]
```

Comment:
I believe this method signature has a problem that results in the block not appearing in the output. I think it should instead be:

`[depth [form [x y z] & block]]`

Notice that the `& block` is now contained within the preceding brackets. I believe that would fix the problem I've had with this, which I've worked around by using the alternate form that takes the angle and vector.